docs/reference/expectations/result_format.md

```diff
@@ -11,6 +11,7 @@ The `result_format` parameter may be either a string or a dictionary which speci
     * `return_unexpected_index_query`: When running validations, a query (or a set of indices) will be returned that will
       allow you to retrieve the full set of unexpected results including any columns identified in `unexpected_index_column_names`.  Setting this value to `False` will 
       suppress the output (default is `True`).
+    * `return_unexpected_index_list`:  Setting this value to `False` will suppress `unexpected_index_list` output (default is `True`).
```

Comment:
Update to documentation

Reply:
Could you provide an example of when a user would want to do this? Is it for cases where a user wants the query but not the list?

Reply:
@talagluck this was a request from a user. I believe they just wanted to use the unexpected results directly, without going through the DataDocs report

Reply:
I sent a link to the thread to Slack